src/pepr/zarf.yaml

```
@@ -32,6 +32,28 @@ variables:
     default: "true"
 
 components:
+  - name: uds-crds
+    required: true
+    charts:
+      - name: uds-cluster-crds
+        namespace: pepr-system
+        version: 0.1.0
+        localPath: uds-cluster-crds
+    actions:
+      onDeploy:
+        before:
+          - description: "Add helm ownership if necessary for clean helm upgrade"
+            mute: true
+            cmd: |
+              # Label the specific CRDs
+              ./zarf tools kubectl label $(./zarf tools kubectl get crds | awk '/exemptions.uds.dev|packages.uds.dev/ {print "crd/" $1}') "app.kubernetes.io/managed-by=Helm" --overwrite || true
+
+              # Annotate with release name
+              ./zarf tools kubectl annotate $(./zarf tools kubectl get crds | awk '/exemptions.uds.dev|packages.uds.dev/ {print "crd/" $1}') "meta.helm.sh/release-name=uds-cluster-crds" --overwrite || true
+
+              # Annotate with release namespace
+              ./zarf tools kubectl annotate $(./zarf tools kubectl get crds | awk '/exemptions.uds.dev|packages.uds.dev/ {print "crd/" $1}') "meta.helm.sh/release-namespace=pepr-system" --overwrite || true
```

This feels overly complex, we should be able to just directly label/annotate two CRDs in a single kubectl call without needing a get/awk in there?
```suggestion
              # Label the specific CRDs
              ./zarf tools kubectl label crd exemptions.uds.dev packages.uds.dev "app.kubernetes.io/managed-by=Helm" --overwrite || true
              # Annotate with release name
              ./zarf tools kubectl annotate crd exemptions.uds.dev packages.uds.dev "meta.helm.sh/release-name=uds-cluster-crds" --overwrite || true
              # Annotate with release namespace
              ./zarf tools kubectl annotate crd exemptions.uds.dev packages.uds.dev "meta.helm.sh/release-namespace=pepr-system" --overwrite || true
```

I think the resulting command ends up being the same unless someone had only one of the CRs on their cluster - but the net result should be the same always due to the `|| true` fallback.